.github/workflows/pr.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.9", "3.10"]
         runs-on: [ubuntu-latest]
         experimental: [false]
         # TODO: Uncomment when netcdf is unpinned
```

Comment:
This is unrelated to this PR per say, but is this TODO something we should act on now that netcdf is unpinned?